code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -31,15 +31,15 @@
 	if(draining)
 		to_chat(src, "<span class='revenwarning'>You are already siphoning the essence of a soul!</span>")
 		return
-	if(target in drained_mobs)
+	if(target.UID() in drained_mobs)
```

Comment:
I'm blind and didn't notice this var was used down the line, my bad. This should go back to how it was.
```suggestion
	var/mob_UID = target.UID()
	if(mob_UID  in drained_mobs)
```